### PR TITLE
Spanish and Catalan taxonomy

### PR DESCRIPTION
**Pull request by AcuarioCat**
Spanish, Catalan and tidy-up

### CODE REVIEW

File: taxonomies/additives.txt [hunk 1]
```diff
@@ -12989,7 +12989,7 @@ cs:E472e, Estery mono- a diglyceridů mastných kyselin s kyselinou mono- a diac
 da:E472e, Mono- og diacetylvinsyreestere af mono- og diglycerider af fedtsyrer
 de:E472e, Mono- und Diacetylweinsäureester von Mono- und Diglyceriden von Speisefettsäuren, Diacetylweinsäureglyceride, Diacetylweinsäureester von Mono- und Diglyceriden von Speisefettsäuren, E 472e
 el:E472e, Εστερες του μονο- και διακετυλοτρυγικου οξεος με μονο- και διγλυκεριδια λιπαρων οξεων
-es:E472e, Ésteres monoacetiltartárico y diacetiltartárico de monoglicéridos y diglicéridos de ácidos grasos, Éster monoacetiltartárico de mono- y diglicéridos de ácidos grasos
+es:E472e, Ésteres monoacetiltartárico y diacetiltartárico de monoglicéridos y diglicéridos de ácidos grasos, Éster monoacetiltartárico de mono- y diglicéridos de ácidos grasos,datem
```
Comment (stephanegigandet) [inline taxonomies/additives.txt:12992]: datem? what does it mean? the only product with it seems to be https://es.openfoodfacts.org/producto/7501000122332/pan-blanco-bimbo

Comment (AcuarioCat) [reply taxonomies/additives.txt:12992]: I wondered too, it's defined here: https://en.wikipedia.org/wiki/DATEM

Comment (stephanegigandet) [reply taxonomies/additives.txt:12992]: Sounds good. Can you put the DATEM entry on the list of English synonyms instead? English synonyms of additives are applied automatically to all languages.

Comment (AcuarioCat) [reply taxonomies/additives.txt:12992]: ok, done. There are entries in en: and fr: as well. Can there be removed?

Comment (stephanegigandet) [reply taxonomies/additives.txt]: Sorry I wasn't clear, I meant putting DATEM in the en: line -> that makes it a synonym.

The lines starting with synonyms:en: are used to make synonym of words used in taxonomy entries.

File: taxonomies/additives.txt [hunk 2]
```diff
@@ -17,6 +17,7 @@ stopwords:en: of
 stopwords:fr: de,d,des,du,le,la,les,a,aux
 
 # synonyms need to be separated with an empty line
+synonyms:en:DATEM, Mono- and diacetyltartaric acid esters of mono- and diglycerides of fatty acids
```
Comment (stephanegigandet) [inline taxonomies/additives.txt:20]:
```suggestion
```

**Pull request merged**